Mathlib/Algebra/MonoidAlgebra/Basic.lean

```diff
@@ -516,6 +526,10 @@ theorem of_injective [MulOneClass G] [Nontrivial k] :
   simpa using (single_eq_single_iff _ _ _ _).mp h
 #align monoid_algebra.of_injective MonoidAlgebra.of_injective
 
+theorem of_commute [MulOneClass G] {a : G} (h : ∀ a', Commute a a') (f : MonoidAlgebra k G) :
+    Commute (of k G a) f :=
```

Comment:
```suggestion
theorem forall_commute_of [MulOneClass G] {a : G} (h : ∀ a', Commute a a') (f : MonoidAlgebra k G) :
    Commute (of k G a) f :=
```
so that the `of` is visibly **not** a lemma name connective

Reply:
I don't agree with these renames, there is already precedence in this file for `of` being used in this way, and `forall` is a meaningless prefix on most lemmas, since they all use a `forall`.